### DISCUSSION TITLE
[ONVIF Device Test Tool 21.12] RTSS-1-1-46 Fail

### DISCUSSION THREAD

**Discussion by truen-scbaek** [onvif/testspecs#21] [Q&A] [answered]
Hello,

RTSS-1-1-46-v21.12 VIDEO ENCODER CONFIGURATION – JPEG RESOLUTION test fails.
Fail log is ...
====================
Waiting 62 frames up to 5000ms
Only 0 frames captured (0.0 FPS)
====================
Test sometimes succeed, but more often fail.
when test failed, I checked that JPEG packet was being sent correctly on Wireshark.
Additionally, There is no problem with the previous version(21.06).

The camera information used for the test is as follows.
Model          : Axis Q3515
F/W Version : 10.8.1

Thanks

**Comment by svefredrik** [comment]
Please post ticket on Axis support location if you think this is an issue with an Axis device. This location is for discussions on ONVIF test tool specs.

**Reply by vinwork** [reply]
I also experience the same issue with RTSS-1-1-46-v21.12 Video Encoder Configuration - JPEG Resolution and RTSS-1-1-46-v21.12 Video Encoder Configuration - H.264 Resolution where it said "Only 0 frames captured (0.0 FPS)" while waiting for 4 frames up to 8000ms while Wireshark received. Increasing Operation delay doesn't help.

There were no problem with previous version 21.06 Onvif Test Tool for the conformance process.

**Comment by madhurao68** [comment, marked as answer]
Errata for tests RTSS-1-1-46-v21.12 VIDEO ENCODER CONFIGURATION – JPEG RESOLUTION and RTSS-1-1-48-v21.12 VIDEO ENCODER  CONFIGURATION – H.264 RESOLUTION for v21.12 released and hosted on the members portal. This issue will be fixed and will be included in the next release of test tool v22.06.
